packages/component/src/Localization/Localize.js

```diff
@@ -1,9 +1,10 @@
 // Localize is designed to be elaboratively return multiple results and possibly exceeding complexity requirement
 /* eslint complexity: "off" */
 
+import connectToWebChat from '../connectToWebChat';
```

Comment:
Web Chat is moving away from the `connectToWebChat` API in favor of hooks. Please use the `useLocalize` hook instead.
```suggestion
```

Reply:
applied , same as above comment